tests/cloud/test_api.py

```diff
@@ -61,6 +61,31 @@ def test_zip_project_ignore_prefixes(tmp_directory, sample_project):
     }
 
 
+def test_zip_project_errors_if_file_too_large(tmp_directory, sample_project):
+    with patch('pathlib.Path.stat') as stat, \
+         pytest.raises(BaseException) as excinfo:
+        stat.return_value.st_size = 6 * 1024 * 1024
+        api.zip_project(force=False,
+                        runid='runid',
+                        github_number='number',
+                        verbose=False)
+    assert "Your project's source code is over 5MB" in str(excinfo.value)
+
+
+def test_upload_zipped_project_errors_if_too_large(tmp_directory):
```

Comment:
This will probably fail until we push the cloud update right?

Reply:
Yes